tools/legacy/BuildMonitor/Legacy_BuildMonitor.csproj

```diff
@@ -194,9 +195,7 @@
       <SubType>Designer</SubType>
     </EmbeddedResource>
   </ItemGroup>
-  <ItemGroup>
-    <Folder Include="..\..\BuildMonitor\Resources\" />
-  </ItemGroup>
```

Comment:
VS2017 did that.  I didn't specifically do this.  I did see a warning in the build output but it didn't block the build.